react-app/src/routes/navigation/navigation.styles.jsx

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+
+export const NavigationContainer = styled.div`
+  height: 70px;
```

Comment:
I'm thinking instead of custom css, could we use any react framework to build the page?

Reply:
sure thing